src/app/loginComponents/accounts/internal/accounts.component.html

```diff
@@ -25,16 +25,22 @@ <h3>Dockstore Profile</h3>
       <mat-list>
         <mat-list-item>
           <h1 matLine>
-            <strong>Username:</strong>
+            <mat-icon>account_circle</mat-icon> <strong class="align-middle">Username:</strong>
           </h1>
           <p matLine id="account-username">{{user?.username}}</p>
         </mat-list-item>
         <mat-list-item>
           <h1 matLine>
-            <strong>Administrator:</strong>
+            <mat-icon>lock</mat-icon> <strong class="align-middle">Administrator:</strong>
           </h1>
           <p matLine id="account-is-admin">{{ user?.isAdmin ? 'Yes' : 'No' }}</p>
```

Comment:
A thought for discussion:
a normal user probably doesn't need to know that they aren't a administrator or that they aren't a curator. 
It only really needs to show up if the logged in user is actually one of the two.
So it can be simplified to just show the icon and the title if you are (kinda like a badge), and just hide it if they aren't.